main/nebula/index.ts

```diff
@@ -2,13 +2,16 @@
 // node-fetch instead of electron-fetch
 const electron = process.versions.electron
 
-// @ts-ignore
-delete process.versions.electron
+type Mutable<T> = {
```

Comment:
I don't think this type of typing really adds any clarity (more the opposite actually), it's really just satisfying the linting. is there another way we can satisfy it? if anything just casting to `any` would be easier and more straightforward imo

Reply:
I thought it was quite elegant 🤷‍♂️   I guess that's a TS usage preference, specifying everything wherever you can vs. more succinct types and `any` usage

Reply:
I was wondering how exactly we are deleting this property if it is readonly and if there another way of forcing `ipfs-utils` to use `node-fetch`...

Presumably we're waiting on [this](https://github.com/ipfs/js-ipfs-utils/issues/140), or the usage of native node fetch.  At that point, this code goes away - so probably we don't care too much about linting it.